ui/components/src/BarChart/BarChart.tsx

```diff
@@ -62,7 +63,7 @@ export function BarChart(props: BarChartProps) {
           source: source,
         },
       ],
-      xAxis: {},
+      xAxis: getYAxes({}, unit),
```

Comment:
@sjcobb Any thoughts on updating `getYAxes` to work for either x or y axis? Could be something like this: 

```
/*
 * Populate yAxis or xAxis properties, returns an Array since multiple axes will be supported in the future
 */
export function formatAxesValues(axis?: YAXisComponentOption | XAXisComponentOption, unit?: UnitOptions) {
  // TODO: support alternate yAxis that shows on right side
  const AXIS_DEFAULT = {
    type: 'value',
    boundaryGap: [0, '10%'],
    axisLabel: {
      formatter: (value: number) => {
        return formatValue(value, unit);
      },
    },
  };
  return [merge(AXIS_DEFAULT, axis)];
}
```

Reply:
Nice yeah I like that!

Reply:
Sweet, updated it!